azure-pipelines.yml

```diff
@@ -173,10 +173,10 @@ jobs:
 #   variables:
```

Comment:
Are these comments necessary? Work to be added later? If they aren't providing any value, I would like to suggest removing them.

Reply:
they are work to add later. I wanted a placeholder so we knew where to put the work once it was done.